pubtools/exodus/_hooks/pulp.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import os
+import sys
+import time
+
+import attr
+import requests
+from pubtools.pluggy import hookimpl, pm  # pylint: disable=wrong-import-order
+from requests.packages.urllib3.util.retry import (  # pylint: disable=import-error
+    Retry,
+)
+from six.moves.urllib.parse import urljoin
+
+LOG = logging.getLogger("pubtools-exodus")
+
+# pylint: disable=unused-argument
+
+
+class ExodusPulpHandler:
+    def __init__(self):
+        self._gw_url = os.getenv("EXODUS_GW_URL")
+        self._gw_env = os.getenv("EXODUS_GW_ENV")
+        self._gw_crt = os.getenv("EXODUS_GW_CERT")
+        self._gw_key = os.getenv("EXODUS_GW_KEY")
+
+        self._retries = os.getenv("EXODUS_GW_RETRIES") or 4
+        self._session = None
+        self._publish = None
+
+    def _exodus_enabled(self):
```

Comment:
Minor - it's up to you, but given that the whole class is private I don't think you really need to name all the methods with leading `_`.